### PR TITLE
negar/add_bem

### PR DESCRIPTION
**Pull request by negarn**
add more BEM

### CODE REVIEW

File: src/javascript/app_2/App/Components/Layout/Header/login-button.jsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Button              from '../../Form/button.jsx';
 
 const LoginButton = ({ className }) => (
     <Button
-        className={classNames(className, 'secondary orange')}
+        className={classNames(className, 'btn-secondary btn--secondary--orange')}
```
Comment (OskarAhl) [inline src/javascript/app_2/App/Components/Layout/Header/login-button.jsx]: `btn-secondary` should be `btn--secondary`?

File: src/javascript/app_2/App/Components/Form/DatePicker/date-picker.jsx [hunk 2]
```diff
@@ -230,7 +230,7 @@ class DatePicker extends React.Component {
             >
                 { this.renderInputField() }
                 <IconCalendar
-                    className={classNames('datepicker__icon datepicker__icon--calendar', {
+                    className={classNames('input-field__icon datepicker__icon datepicker__icon--calendar', {
```
Comment (OskarAhl) [inline src/javascript/app_2/App/Components/Form/DatePicker/date-picker.jsx]: `input-field__icon `?

Comment (negarn) [reply src/javascript/app_2/App/Components/Form/DatePicker/date-picker.jsx]: these icons that project inside the input field have common styling. I can rename it and move it out of input-field.scss if you think that's better.

Comment (OskarAhl) [reply src/javascript/app_2/App/Components/Form/DatePicker/date-picker.jsx]: Where is the parent block `input-field`?

Comment (negarn) [reply src/javascript/app_2/App/Components/Form/DatePicker/date-picker.jsx]: there's no parent, it's a sibling. in the old code it was handled with a +

Comment (OskarAhl) [reply src/javascript/app_2/App/Components/Form/DatePicker/date-picker.jsx]: Oh I see, would it be less coupled if we added a mixin for this common icon style instead?

Comment (negarn) [reply src/javascript/app_2/App/Components/Form/DatePicker/date-picker.jsx]: sure, I will do that instead.

**Pull request merged**